### PR TITLE
New Indicator Field - Key Value

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/ReleaseNotes/3_3_2.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+#### Indicator Fields
+- **Key Name**
```
Comment (Ni-Knight) [inline Packs/CommonTypes/ReleaseNotes/3_3_2.md:3]: I think "Key Values" is a better name as in win registry:
name\key - "Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Zoom\MSI"
data\values - everything else

Also see:
image

File: Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            "required": false,
+            "script": "",
+            "width": 150,
+            "isDefault": true,
+            "fieldCalcScript": "",
+            "isReadOnly": false,
+            "selectValues": null
+        },
+        {
+            "key": "type",
```
Comment (Ni-Knight) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json:53]: עדיף Type לפני ה-DATA כמו שזה בעורך עצמו
image

**Review by Ni-Knight — CHANGES_REQUESTED** [review]
A few comments.

File: Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json (hunk 2 above)
Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json:53]: We should also put `Type` as a single select with these values:
`REG_NONE,REG_SZ,REG_EXPAND_SZ,REG_BINARY,REG_DWORD,REG_DWORD_LITTLE_ENDIAN,REG_DWORD_BIG_ENDIAN,REG_LINK,REG_MULTI_SZ,REG_RESOURCE_LIST,REG_FULL_RESOURCE_DESCRIPTOR,0REG_RESOURCE_REQUIREMENTS_LIST,1REG_QWORD,REG_QWORD_LITTLE_ENDIAN`

File: Packs/CommonTypes/ReleaseNotes/3_3_2.md (hunk 1 above)
Comment (MLainer1) [reply Packs/CommonTypes/ReleaseNotes/3_3_2.md:3]: No problem, will change it in the design

File: Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            "type": "singleSelect",
+            "orgType": "singleSelect",
+            "required": false,
+            "script": "",
+            "width": 150,
+            "isDefault": true,
+            "fieldCalcScript": "",
+            "isReadOnly": false,
+            "selectValues": [
+                "",
```
Comment (Ni-Knight) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json:51]: Is the empty string on purpose?

**Review by Ni-Knight — APPROVED** [review]
@MLainer1 - Done, just a small comment

File: Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json (hunk 3 above)
Comment (MLainer1) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-keyname.json:51]: Probably not, will remove

**Pull request merged**